src/index.js

```diff
@@ -100,42 +102,45 @@ sequelize.sync({ force: isTest || isProduction }).then(async () => {
 });
 
 const createUsersWithMessages = async date => {
-  await models.User.create(
+  try {
+  const user1 = await models.User.create(
     {
       username: 'rwieruch',
       email: 'hello@robin.com',
       password: 'rwieruch',
       role: 'ADMIN',
-      messages: [
-        {
+
+    },
+  ).then(createdUser => {
```

Comment:
Out of curiosity: Is there no shorthand implementation to create a user with an associated list of messages similar to the Sequelize way of doing it?

Reply:
I actually spent a lot of time finding a shorthand( using Object Literal)  but Nah. there is no shorthand method as Mongodb is not Relational. we would have wait for to be created to us assign a relation